lib/ui/profile/interactor/profile_bloc.dart

```diff
@@ -73,9 +82,27 @@ class ProfileBloc extends Bloc<ProfileEvent, ProfileState> {
     add(const ProfileEvent.initial());
   }
 
+  FutureOr<Result<bool, HyphaError>> _checkLogin() async {
+    final accountName = state.profileData!.account;
+    final res = await _profileLoginUseCase.run(accountName, signUp: state.doesNotHaveProfile);

```

Comment:
doesNotHaveProfile => hasProfile

Reply:
right but it's more of an exceptional case to not have one. Still you're probably right, it was hasProfile initially ;)